src/components/data-table-v2/_data-table-v2-sort.scss

```diff
@@ -66,3 +66,118 @@
     min-width: 14px;
   }
 }
+
+@mixin data-table-v2-sort--x {
+  // -------------------------------------
+  // Sortable table
+  // -------------------------------------
+
+  .#{$prefix}--data-table-v2--sort th:hover {
+    background: $data-table-column-hover;
+    border-top: 1px solid $data-table-column-hover;
+    border-bottom: 1px solid $data-table-column-hover;
+  }
+
+  .#{$prefix}--data-table-v2--sort th.#{$prefix}--table-column-icon:hover,
+  .#{$prefix}--data-table-v2--sort th.#{$prefix}--table-column-menu:hover,
+  .#{$prefix}--data-table-v2--sort th.#{$prefix}--table-expand-v2:hover {
+    background: $ui-03;
+    border-top: 1px solid $ui-03;
+    border-bottom: 1px solid $ui-03;
+  }
+
+  .#{$prefix}--data-table-v2--sort th {
+    padding: 0;
+  }
+
+  .#{$prefix}--data-table-v2--sort th.#{$prefix}--table-column-checkbox {
+    padding: rem(10px) $spacing-05;
+  }
+
+  // -------------------------------------
+  // Th > Button
+  // -------------------------------------
+  .#{$prefix}--table-sort-v2 {
+    @include button-reset(false);
+    position: relative;
+    font: inherit;
+    cursor: pointer;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    color: $text-01;
+    padding: 0 $spacing-04;
+  }
+
+  .#{$prefix}--table-sort-v2:focus {
+    outline: 0;
+  }
+
+  .#{$prefix}--table-sort-v2:focus svg,
+  .#{$prefix}--table-sort-v2:hover svg {
+    opacity: 1;
+  }
+  // -------------------------------------
+  //Th > Button > Svg (Sort Icons)
+  // -------------------------------------
+
+  .#{$prefix}--data-table-v2 th:hover .#{$prefix}--table-sort-v2 .#{$prefix}--table-sort-v2__icon {
+    opacity: 1;
+  }
+
+  // inactive icons
+  .#{$prefix}--table-sort-v2 .#{$prefix}--table-sort-v2__icon-inactive {
+    display: block;
+  }
+
+  .#{$prefix}--table-sort-v2 .#{$prefix}--table-sort-v2__icon {
+    display: none;
```

Comment:
Should we include this alongside the other selectors that are making these icons visually hidden?

Reply:
There aren't any other selectors that make this visually hidden unless we want to do
`.#{$prefix}--table-sort-v2:not(.#{$prefix}--table-sort-v2--active) .#{$prefix}--table-sort-v2__icon`. I originally had it like that but since you said the `:not`'s were confusing in the last PR, I removed them.

Reply:
It's more so I just noticed in the style inspector that `opacity: 1` was being defined three times for the `bx--table-sort-v2__icon`:

![image](https://user-images.githubusercontent.com/3901764/54833165-937b3080-4c8b-11e9-9ff0-5af176855e9e.png)

Just thought it could be a good place to refactor 👍

Reply:
Removed one of the redundant `opacity :1` styles and the left comments for the other 2 explaining why they're needed. One is used for when the `th` doesn't have the active class and only gives that opacity when it's being hovered, and one sets the opacity when it's activated, so it'll always be visible whether hovering or not. Hope that makes sense.